### PR TITLE
fix: bump node 12 to node 18

### PR DESCRIPTION
**Pull request by adong**
## Context

Since node 12 is deprecated, so bump up container to node ~14~ 18

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by pritamstyz4ever — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -262,7 +262,7 @@ module.exports = function configParser({
                 jobs: {
                     main: [
                         {
-                            image: 'node:12',
+                            image: 'node:14',
```
Comment (jithine) [inline index.js]: let's use the latest lts ?

**Comment by tkyi** [comment]
Why was there no build run?

**Comment by tkyi** [comment]
Weird, ended up Syncing Pull Requests on the SD pipeline options page to get the PR build to start.
Need to fix tests: https://cd.screwdriver.cd/pipelines/18/builds/836303/steps/test

**Comment by adong** [comment]
> Weird, ended up Syncing Pull Requests on the SD pipeline options page to get the PR build to start. Need to fix tests: [cd.screwdriver.cd/pipelines/18/builds/836303/steps/test](https://cd.screwdriver.cd/pipelines/18/builds/836303/steps/test)

Good call. `npm run test` and tests are now fixed, please take another look.

**Review by tkyi — APPROVED** [review]

**Pull request merged**